src/Microsoft.AspNetCore.WebUtilities/HttpResponseStreamWriter.cs

```diff
@@ -16,11 +16,7 @@ namespace Microsoft.AspNetCore.WebUtilities
     public class HttpResponseStreamWriter : TextWriter
     {
         private const int MinBufferSize = 128;
-
-        /// <summary>
-        /// Default buffer size.
-        /// </summary>
-        public const int DefaultBufferSize = 1024;
+        internal const int DefaultBufferSize = 16 * 1024;
```

Comment:
I would have, but its value is also used in the FlushesBuffer_ButNotStream_OnFlushAsync test method body and is apparently significant to how the test runs.